printer_mods/README.md

```diff
@@ -268,6 +268,7 @@ like so:
 |madros |[V0 OLED-skirt](./madros/V0_skirt_OLED) |V0 skirt with 0.96" OLED|:heavy_check_mark: |:x: |:x: |:x:|
 |MarcPot |[UHP PSU Bracket Clip](./MarcPot/UHP_PSU_Bracket_Clip)|DIN rail mount for the Mean Well UHP series PSU. |:grey_question: |:grey_question: |:heavy_check_mark: |:x:|
 | |[Hour Counter Skirt + Keystone Skirt](./MarcPot/Skirt_Mods)|Skirt mod for a mechanical hour counter and keystone inserts | :x: |:x: |:heavy_check_mark: |:x:|
+|Mastah |[SexyPlate](./Mastah/sexyplate/)|Steel sheet plate holder on the side of the printer |:grey_question: |:grey_question: |:heavy_check_mark: |:grey_question: | :grey_question: |
```

Comment:
I don't see any reason why this wouldn't work on a Trident or 1.8 as well, those can probably get the ✔️

Reply:
Can't test so not really whiling to put check mark there.
If people test and actually give a go for those printer, happy to add them in readme.